src/scripts/OSFramework/OSUI/Pattern/OverflowMenu/Enum.ts

```diff
@@ -0,0 +1,41 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.OverflowMenu.Enum {
+	/**
+	 * OverflowMenu Enum AriaLabel
+	 */
+	export enum AriaLabel {
+		Trigger = 'Trigger the balloon',
```

Comment:
@BenOsodrac We should start to provide a way to change this since the developer might want to change the text and/or to allow translations.

Reply:
Added a public method to update it 👍